### ISSUE TITLE
DO_JUMP not displaying correctly on map

### ISSUE THREAD

**Issue by tridge** [ArduPilot/MissionPlanner#2912]
using DO_JUMP is normal in most fixed wing missions, but MissionPlanner doesn't display it correctly
here is a MissionPlanner display of a typical quadplane mission:
![image](https://user-images.githubusercontent.com/831867/180087725-5116ebd2-a888-4d02-9fee-4959d688a3d7.png)
here is the same mission in MAVProxy:
![image](https://user-images.githubusercontent.com/831867/180087826-33ab4053-6bf4-4c22-8a3d-56d7ce12e9bb.png)
notice the correct line for the segment following the DO_JUMP, and the correct display of the DO_LAND_START
the mission file for this example is attached
Thanks!
[wp.txt](https://github.com/ArduPilot/MissionPlanner/files/9154680/wp.txt)

**Comment by rmackay9** [comment]
Ah, so if there is a do-jump then the wp before the do-jump needs to display two lines.  One to the command index from the do-jump command and one to the command after the do-jump.

**Comment by tridge** [comment]
> Ah, so if there is a do-jump then the wp before the do-jump needs to display two lines. One to the command index from within the do-jump command and one to the command after the do-jump.

no, I think we should only display the "jump is taken" case. That is what MAVProxy has always done and it has worked very well and is very easy to understand
